Problem with omada script

### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

getting error:

`[ERROR] in line 33: exit code 8: while executing command wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb`

checking the url there is no ver. 2.19 anymore but new 2.20

`[libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb](http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb)`


### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

_No response_

### Please provide detailed steps to reproduce the issue.

executing bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/omada.sh)" in proxmox shell

--- a/install/deconz-install.sh
+++ b/install/deconz-install.sh
@@ -26,8 +26,8 @@ echo "deb [arch=amd64] http://phoscon.de/apt/deconz $VERSION main" >/etc/apt/sou
 msg_ok "Setup Phoscon Repository"
 
 msg_info "Installing deConz"
-wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
-$STD dpkg -i libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
+wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb
+$STD dpkg -i libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb
 $STD apt-get update
 $STD apt-get install -y deconz
 msg_ok "Installed deConz"
@@ -56,7 +56,7 @@ motd_ssh
 customize
 
 msg_info "Cleaning up"
-rm -rf libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
+rm -rf libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb
 $STD apt-get autoremove
 $STD apt-get autoclean
 msg_ok "Cleaned"

--- a/install/unifi-install.sh
+++ b/install/unifi-install.sh
@@ -31,8 +31,8 @@ $STD apt-mark hold openjdk-11-*
 msg_ok "Installed OpenJDK"
 
 msg_info "Installing MongoDB"
-wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
-$STD dpkg -i libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb
+wget -qL http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb
+$STD dpkg -i libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb
 wget -qL https://repo.mongodb.org/apt/ubuntu/dists/bionic/mongodb-org/3.6/multiverse/binary-amd64/mongodb-org-server_3.6.23_amd64.deb
 $STD dpkg -i mongodb-org-server_3.6.23_amd64.deb
 msg_ok "Installed MongoDB"
@@ -48,7 +48,7 @@ motd_ssh
 customize
 
 msg_info "Cleaning up"
-rm -rf mongodb-org-server_3.6.23_amd64.deb libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb /etc/apt/sources.list.d/openjdk-11-jre-headless.list
+rm -rf mongodb-org-server_3.6.23_amd64.deb libssl1.1_1.1.1f-1ubuntu2.20_amd64.deb /etc/apt/sources.list.d/openjdk-11-jre-headless.list
 $STD apt-get autoremove
 $STD apt-get autoclean
 msg_ok "Cleaned"

